src/main/java/io/arivera/oss/embedded/rabbitmq/EmbeddedRabbitMqConfig.java

```diff
@@ -446,9 +456,14 @@ public EmbeddedRabbitMqConfig build() {
           erlangCheckTimeoutInMillis,
           cacheDownload, deleteCachedFile,
           envVars,
-          processExecutorFactory);
+          processExecutorFactory,
+          downloadProxy);
     }
 
+    public Builder downloadProxy( String hostname, int port ) {
```

Comment:
Also, please move this method _above_ the actual `build()` method.

It's generally a good practice to order the methods in the class in the order that you'd expect them to be called and/or read. "Clean Code" refers this to as: "Clean code reads like well-written prose." and Uncle Bob (the author) also recommends following the "[step down rule](https://grysz.com/2015/11/13/reformat-java-code-in-intellij-according-to-uncle-bobs-stepdown-rule/)".